ml-proto/README.md

```diff
@@ -157,9 +145,15 @@ expr:
   ( memory_size )
   ( grow_memory <expr> )
 
+switch:
+  ( table <target>* )
```

Comment:
Part of me was hoping we could use a LISP-style array literal, `#( <target>* )` or `[ <target>* ]` depending on the dialect, but I'm ok if we can't because of generic S-expression parsers.